examples/docker-mixin-test/porter.yaml

```diff
@@ -37,6 +43,14 @@ install:
       command: cowsay
       arguments:
         - "Hello World"
+- docker:
+    description: "Login to docker"
```

Comment:
I'm going to look into if I can make description optional for mixins to implement. I think it may work. 🤔 Then it could be slimmer 

```yaml
- docker: login
```